sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/OrchestrationService.java

```diff
@@ -53,6 +62,45 @@ public OrchestrationChatResponse completion(@Nonnull final String famousPhrase)
     return client.chatCompletion(prompt, config);
   }
 
+  /**
+   * Chat request to OpenAI through the Orchestration service with an image.
+   *
+   * @return the assistant response object
+   */
+  @Nonnull
+  public OrchestrationChatResponse imageInput(@Nonnull final String pathToImage) {
+    final OrchestrationAiModel customGpt4o =
+        GPT_4O_MINI
+            .withParam(MAX_TOKENS, 50)
+            .withParam(TEMPERATURE, 0.1)
+            .withParam(FREQUENCY_PENALTY, 0)
+            .withParam(PRESENCE_PENALTY, 0)
+            .withParam(TOP_P, 1)
+            .withParam(N, 1);
```

Comment:
unless we really need this for this test case, I'd rather keep the focus of the sample code on the image input